parser-typechecker/src/Unison/Codebase/SqliteCodebase/Migrations/MigrateSchema1To2.hs

```diff
@@ -50,6 +50,7 @@ import qualified U.Codebase.Sqlite.Patch.Full as S
 import qualified U.Codebase.Sqlite.Patch.TermEdit as TermEdit
 import qualified U.Codebase.Sqlite.Patch.TypeEdit as TypeEdit
 import qualified U.Codebase.Sqlite.Queries as Q
+import U.Codebase.Sqlite.V2.HashHandle (v2HashHandle)
```

Comment:
Note: we realize this hash handle isn't actually going to be used in the migration. (none of `saveCausal`/`saveDbBranchUnderHashId`/`saveDbPatch` use it)